src/test/java/org/jabref/logic/formatter/FormatterTest.java

```diff
@@ -38,102 +38,101 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class FormatterTest {
+class FormatterTest {
 
     private static ProtectedTermsLoader protectedTermsLoader;
 
     @BeforeAll
-    public static void setUp() {
+    static void setUp() {
         protectedTermsLoader = new ProtectedTermsLoader(
                 new ProtectedTermsPreferences(ProtectedTermsLoader.getInternalLists(), Collections.emptyList(),
                         Collections.emptyList(), Collections.emptyList()));
 
+    }
+
+    private static Stream<Formatter> getFormatters() {
+        // all classes implementing {@link net.sf.jabref.model.cleanup.Formatter}
+        // sorted alphabetically
+        // Alternative: Use reflection - https://github.com/ronmamo/reflections
+        // @formatter:off
+       return Stream.of(
+                new CapitalizeFormatter(),
+                new ClearFormatter(),
+                new HtmlToLatexFormatter(),
+                new HtmlToUnicodeFormatter(),
+                new IdentityFormatter(),
+                new LatexCleanupFormatter(),
+                new LatexToUnicodeFormatter(),
+                new LowerCaseFormatter(),
+                new MinifyNameListFormatter(),
+                new NormalizeDateFormatter(),
+                new NormalizeMonthFormatter(),
+                new NormalizeNamesFormatter(),
+                new NormalizePagesFormatter(),
+                new OrdinalsToSuperscriptFormatter(),
+                new ProtectTermsFormatter(protectedTermsLoader),
+               new RegexFormatter("(\" \",\"-\")"),
+                new RemoveBracesFormatter(),
+                new SentenceCaseFormatter(),
+                new TitleCaseFormatter(),
+                new UnicodeToLatexFormatter(),
+                new UnitsToLatexFormatter(),
+                new UpperCaseFormatter());
 
+        // @formatter:on
     }
 
     @ParameterizedTest
     @MethodSource("getFormatters")
-    public void getNameReturnsNotNull(Formatter formatter) {
+    void getNameReturnsNotNull(Formatter formatter) {
```

Comment:
Why do you remove the public?
Codacy or whatever will complain that you should always use explicit scoping modifiers. I see no harm in using public here.  I don't know if this even has to be public to be accessible from gradle

Reply:
It's the new convention with JUnit 5 that test classes and their methods are package private. IntellJ even complains if they are marked with public.

Reply:
Ah okay, that was new to me